pkgs/development/python-modules/django-sql-utils/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  hatchling,
+  django,
+  sqlparse,
+}:
+
+buildPythonPackage rec {
+  pname = "django-sql-utils";
+  version = "0.7.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "martsberger";
+    repo = "django-sql-utils";
+    tag = version;
+    hash = "sha256-OjKPxoWYheu8UQ14KvyiQyHISAQjJep+N4HRc4Msa1w=";
+  };
+
+  postPatch = ''
+    echo -e "\n[tool.hatch.build.targets.wheel]\npackages = [ \"sql_util\" ]" >> pyproject.toml
+  '';
+
+  build-system = [ hatchling ];
+
+  dependencies = [
+    django
+    sqlparse
+  ];
+
+  checkPhase = ''
+    runHook preCheck
+
+    python runtests.py
```

Comment:
Can we use pytestCheckHook with pytest-django and setting the DJANGO_SETTINGS_MODULE env to sql_util.tests.test_sqlite_settings ?